code/game/objects/machinery/cryopod.dm

```diff
@@ -1,3 +1,12 @@
+#define HS_CNSL_GUN 0
+#define HS_CNSL_AMMO 1
+#define HS_CNSL_EXPLOSIVE 2
+#define HS_CNSL_MELEE 4
+#define HS_CNSL_CLOTHING 5
+#define HS_CNSL_FOOD 6
+#define HS_CNSL_DRUGS 7
+#define HS_CNSL_CONTAINERS 8
+#define HS_CNSL_OTHER 9
```

Comment:
Don't forget to ``#undef`` your defines at the end of the file or else they are global.